### PR TITLE
Fix(DB/Scripts): changed event param to Bodley' script

### PR DESCRIPTION
**Pull request by Nefertumm**
## Changes Proposed:
-  The event_param2 was mixed with those ids, giving the wrong item.
-  

## Issues Addressed:

- Closes 

## Tests Performed:

- Tested in-game. https://www.youtube.com/watch?v=K1QVSW1LxMo

## How to Test the Changes:

1. go c id 16033
2. Test the multiple gossips from Bodley with certain conditions

### CODE REVIEW

**Comment by Shineslippers** [comment]
Tested:

- [X] Possiblity to recover Left Piece of Lord Valthalak's Amulet
- [X] Possiblity to recover Top Piece of Lord Valthalak's Amulet
- [ ] Possiblity to recover both pieces at one conversation
- [X] Possiblity to recover Lord Valthalak's Amulet

https://user-images.githubusercontent.com/85256266/146816160-c67f83f6-81c8-4b63-920d-032231f78f64.mp4

**Comment by Nefertumm** [comment]
@Shineslippers the gossip get stuck because of the smart_script (I think), because once you select the gossip, the menu should change, that's why you can't recover 2 items in one talk. I think is not related to my pr.

**Review by Nyeriah — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1640023074644304400.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1640023074644304400');
+
+UPDATE `smart_scripts` SET `event_param2` = 1 WHERE `entryorguid` = 16033 AND `source_type` = 0 AND `id` = 0;
+UPDATE `smart_scripts` SET `event_param2` = 0  WHERE `entryorguid` = 16033 AND `source_type` = 0 AND `id` = 1;
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1640023074644304400.sql:4]:
```suggestion
UPDATE `smart_scripts` SET `event_param2` = 0 WHERE `entryorguid` = 16033 AND `source_type` = 0 AND `id` = 1;
```

**Comment by Shineslippers** [comment]
https://user-images.githubusercontent.com/85256266/147287820-855c4831-2a82-48b7-a901-527047718d58.mp4

**Pull request merged**